packages/js/src/account/account-manager.ts

```diff
@@ -217,7 +216,7 @@ export abstract class AccountManager implements NearAccountManager {
   }
 
   protected get connection(): nearAPI.Connection {
-    return new nearAPI.Connection(this.networkId, this.provider, this.signer);
+    return new nearAPI.Connection(this.networkId, this.provider, this.signer, `jsvm.${this.networkId}`);
```

Comment:
Is this added by mistake?

Reply:
No, constructor of `Connection` changed: https://github.com/near/near-api-js/blob/master/packages/accounts/src/connection.ts#L41